components/tools/OmeroWeb/omeroweb/http.py

```diff
@@ -35,12 +34,6 @@ def __init__(self, content):
             self, content, content_type="text/javascript")
```

Comment:
Since there's no built-in Django response that has the 500 status code and 'javascript' as the content_type, we still need to create our own for this.